src/data.js

```diff
@@ -19,9 +19,6 @@ var Hook = Entity.configure({
     task:               Entity.types.JSON,
     // pulse bindings (TODO; empty for now)
     bindings:           Entity.types.JSON,
-    // timings for the task (in fromNow format, e.g., "1 day")
-    deadline:           Entity.types.String,
-    expires:            Entity.types.String,
```

Comment:
For changes to the Entity schema, we need to add a new version to the list of `configure({version: ..}).configure({version: ..})` here.  That version should no longer list these two fields, and its `migrate` function should delete them.  We can't change the definition of the old versions because there may still be data of that version out there.